crates/db_schema/src/impls/actor_language.rs

```diff
@@ -612,8 +586,8 @@ mod tests {
     LocalUserLanguage::update(pool, test_langs3, local_user.id).await?;
 
     // this time, both have ru as common lang
-    let def2 = validate_post_language(pool, None, community.id, local_user.id).await?;
-    assert_eq!(ru, def2);
+    let def2 = validate_post_language(pool, None, community.id).await;
+    assert!(def2.is_ok());
```

Comment:
Shouldn't this still check for ru just to be sure?

Reply:
Previously `validate_post_language()` would return the language for the post (eg if the user only has a single language active). Now it only checks if the given language is valid and doesnt return anything, so theres nothing to check.